mkosi/run.py

```diff
@@ -366,6 +366,13 @@ def bwrap(
     **kwargs: Any,
 ) -> CompletedProcess:
     with bwrap_cmd(root=root, apivfs=apivfs) as bwrap:
+        if root:
+            # If a root is specified, we should ignore any local modifications made to PATH as any of those
+            # tools might not work anymore when /usr is replaced wholesale. We also make sure that both
```

Comment:
Is the sentence starting with "We also make sure" missing a part, i.e. what it is we're making sure?